systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -1413,14 +1378,7 @@ void verifySinkConnectorByBlockAndUnblock(TestStorage testStorage, String kafkaC
 
         // messages need to be produced for each run (although there are more messages in topic eventually, sink will not copy messages it copied previously (before clearing them)
         LOGGER.info("Producing new messages which are to be watched KafkaConnector once it is resumed");
-        final KafkaClients kafkaClients = new KafkaClientsBuilder()
-            .withTopicName(testStorage.getTopicName())
-            .withMessageCount(testStorage.getMessageCount())
-            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
-            .withProducerName(testStorage.getProducerName())
-            .withConsumerName(testStorage.getConsumerName())
-            .withNamespaceName(testStorage.getNamespaceName())
-            .build();
+        final KafkaClients kafkaClients = ClientUtils.getDefaultBasicClientBuilder(testStorage).build();
```

Comment:
as I mentioned offline, I don't really like that everything is completely hidden under `getDefaultBasicClientBuilder`
especially the `withBootstrapAddress()` - what if you change the default bootstrap address inside the `getDefaultBasicClientBuilder` to be TLS? I think that at least the bootstrap address should be added directly here, so it is more clear (and also you will be sure that in case of change in `getDefaultBasicClientBuilder` you will not break all the tests)